tests/commands/probes.py

```diff
@@ -145,8 +145,7 @@ def test_location_arg(self):
                     {"geometry": {"location": {"lat": 1, "lng": 2}}}]}
                 cmd = Command()
                 cmd.init_args(["--location", "blaaaa"])
-                self.assertEquals(cmd.build_request_args(), {
-                    "latitude": '1', "longitude": '2'})
+                self.assertEquals(cmd.build_request_args(), {'radius': '1,2:15'})
```

Comment:
Shouldn't this be `{"radius": "15", "center": "1,2"}`?  Also, how the hell did we settle on American spelling for *centre*?